src/subhalo_update_mostbound.cpp

```diff
@@ -40,6 +40,12 @@ void SubhaloSnapshot_t::UpdateMostBoundPosition(MpiWorker_t &world, const Partic
       ZeroSizeSubhalo[nr_zero] = sub;
       ZeroSizeSubhalo[nr_zero].Particles.resize(1);
       ZeroSizeSubhalo[nr_zero].Particles[0] = Particle_t(sub.MostBoundParticleId);
+      for (int j = 0; j < 3; j += 1)
```

Comment:
Good to keep, but this should not be triggered since all our tracers are time-persistent and so if one is missing, something has gone wrong!